src/main/java/org/apache/jackrabbit/filevault/maven/packaging/VaultMojo.java

```diff
@@ -585,7 +599,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
             vaultDir.mkdirs();
 
-            Map<String, File> embeddedFiles = copyEmbeddeds();
+            Map<String, PlexusIoResource> embeddedFiles = copyEmbeddeds();
```

Comment:
This is necessary to be able to include input streams in the resulting zip. Otherwise one would need to copy from the class loader resource to a temporary file before including in the zip